save-cloud-common/src/jvmMain/kotlin/com/saveourtool/save/entities/TestSuite.kt

```diff
@@ -35,7 +39,30 @@ class TestSuite(
     var language: String? = null,
 
     var tags: String? = null,
+
+    var plugins: String = ""
```

Comment:
why `var`?

Reply:
It is always so in Entities. Moreover, I use setters to update the value of it